app/assets/javascripts/map/components/registration-form.js

```diff
@@ -52,9 +58,14 @@ function RegistrationForm() {
           type: 'text',
           name: 'email',
           value: data.email,
-          onchange: event => { data.email = event.currentTarget.value },
+          // onchange: event => { data.email = event.currentTarget.value },
```

Comment:
I believe this needs to be uncommented in order for the email to be submitted correctly.

Could you test that you are still able to register with a valid email. If you register successfully, then your registration will appear in the "Attendees" tab of the event in the cms.

Reply:
![image](https://user-images.githubusercontent.com/26555558/234061588-79e7e0c3-bd30-49a0-b248-1316117fb816.png)
able to register